warehouse/templates/pages/help.html

```diff
@@ -26,6 +26,7 @@
 {% macro verified_email() %}Why do I need a verified email address?{% endmacro %}
 {% macro mirroring() %}How can I run a mirror of PyPI?{% endmacro %}
 {% macro APIs() %}Does PyPI have APIs I can use?{% endmacro %}
+{% macro notifications() %}How do I get notified when a new version of a package is released?{% endmacro %}
```

Comment:
We try to use the word "project" where you say "package" here.

Reply:
Could you use a more specific function name and anchor tag label? I want to futureproof here a bit; probably we will have a future FAQ item about project owners'/maintainers' email notifications.